### ISSUE TITLE
Can you provide the PDB file of the test set?

### ISSUE THREAD

**Issue by YangW-Labs** [flatironinstitute/DeepFRI#52]
Hi,

Thanks for opensource this wonderful work. The link provided by the project contains only the training dataset and validation set data. Can you provide the PDB file of the test set?

**Comment by tahmid1234** [comment]
> Hi,
> 
> Thanks for opensource this wonderful work. The link provided by the project contains only the training dataset and validation set data. Can you provide the PDB file of the test set?

Hi have you found the test dataset? If you have, could you help me find it?
